--- php.ini	2022-11-24 13:30:28.602149109 -0300
+++ p2.ini	2022-11-24 13:31:11.146300988 -0300
@@ -406,7 +406,7 @@
 ; Maximum execution time of each script, in seconds
 ; https://php.net/max-execution-time
 ; Note: This directive is hardcoded to 0 for the CLI SAPI
-max_execution_time = 30
+max_execution_time = 60
 
 ; Maximum amount of time each script may spend parsing request data. It's a good
 ; idea to limit this time on productions servers in order to eliminate unexpectedly
@@ -427,7 +427,7 @@
 
 ; Maximum amount of memory a script may consume
 ; https://php.net/memory-limit
-memory_limit = 128M
+memory_limit = 512M
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ; Error handling and logging ;
@@ -924,7 +924,7 @@
 [Date]
 ; Defines the default timezone used by the date functions
 ; https://php.net/date.timezone
-;date.timezone =
+date.timezone = 'America/Sao_Paulo'
 
 ; https://php.net/date.default-latitude
 ;date.default_latitude = 31.7667

